schedule/yast/remote_vnc/remote_vnc_controller_opensuse.yaml

```diff
@@ -2,21 +2,24 @@
 name:           remote_vnc_controller
```

Comment:
similar for this (check rest of the files just in case), so if doesn't work yet the networking, let's exclude it for now and create a ticket with low prio.